pkg/auth/github/provider.go

```diff
@@ -71,6 +80,16 @@ func (p *Provider) GetUserDetails(code string, user *auth.User) error {
 		}
 	}
 
+	if p.Teams != "" {
+		if p.Organization == "" {
+			return fmt.Errorf("organization is required for teams")
+		}
+
+		if err := p.PerformCheckUserMemberOfTeams(t); err != nil {
+			return err
+		}
+	}
```

Comment:
There is no need for this validation, as it is handled by the configuration.

```suggestion
	if p.Teams != "" {
		if err := p.PerformCheckUserMemberOfTeams(t); err != nil {
			return err
		}
	}
```

Reply:
Forgot to push the file :(